app/presenters/job_presenter.rb

```diff
@@ -2,7 +2,7 @@ module JobPresenter
   class << self
     def single(job)
       payload = job['payload']
-      return if job['payload'].blank? || payload['created_from'] != 'API'
+      return if (job['payload'].blank? || payload['created_from'] != 'API') && (job['payload'].blank? || payload['created_from'] != 'BackendFlow')
```

Comment:
Do you mean:
```ruby
return if job['payload'].blank? || !['API', 'BackendFlow'].include?(payload['created_from'])
```

Also, your commit message talks about firing jobs. This code change will not fire any jobs, this will just show jobs in the job listing API that have `created_from = BackendFlow`.

Reply:
I have updated the PR according to the above comment.